node/src/accountant/mod.rs

```diff
@@ -1862,13 +1882,21 @@ mod tests {
             from: make_wallet("wallet1"),
             wei_amount: 10000,
         };
-        let more_money_received_params_arc = Arc::new(Mutex::new(vec![]));
+        let transaction_id = ArbitraryIdStamp::new();
+        let transaction = TransactionWrapperMock::default()
```

Comment:
Maybe rename this variable to `transaction_wrapper` to avoid confusion.

Reply:
To me, the fact it is a transaction wrapped in something doesn't change that it is still the transaction inside what matters. And because a wrapper by itself is not supposed to add functionality I consider these two terms interchangeable.